frontend/public/components/monitoring.tsx

```diff
@@ -801,7 +801,7 @@ class SilenceForm_ extends SafetyFirst<SilenceFormProps, SilenceFormState> {
         <div className="form-group">
           <label>Matchers</label> (label selectors)
           <p className="co-help-text">Alerts affected by this silence. Matching alerts must satisfy all of the specified label constraints, though they may have additional labels as well.</p>
-          <div className="row co-m-table-grid__head">
+          <div className="row monitoring-grid-head">
             <div className="col-xs-4">Name</div>
```

Comment:
`<div className="col-xs-4 text-secondary text-uppercase">Name</div>`

Reply:
I put `text-secondary text-uppercase` on the `row` rather than the 2 `col-xs-4` cells, but can change that if you feel strongly that they belong on the individual heading cells.